ui/src/app/components/resources/dataset-action-menu.tsx

```diff
@@ -96,7 +96,7 @@ export const DatasetActionMenu = fp.flow(
               resource.workspaceFirecloudName,
               'data',
               'data-sets',
-              resource.dataSet.id,
```

Comment:
several data-apps `id`s are used as both `number` and `string` depending on context.  Here, it was not an error because it was untyped, and it performed an implicit conversion.